app/main/util/date_validator.py

```diff
@@ -101,85 +116,116 @@ def validate_date(  # noqa: C901
     check_future_date=True,
 ):
     errors = []
+    error_field = (
```

Comment:
Instead of removing the formatting this here, let's just change what we pass in.

Instead of passing in "`Date from`", pass in "date_from"

Reply:
just using the existing input of "’Date from’" to turn the value in "date_from" , no new parameter has been passed, if you still insist , we need to pass another parameter to validate_date function, please confirm and I can do the changes

Reply:
my point is that it is simpler to just change the original we pass into the function as it is only used for the purpose of this error message and error field

Reply:
clarified on call, all good